skripchenkoveta/hometask-4/spec/my_hash_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require '../my_hash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  before do
+    my_hash['li'] = 'za'
+    my_hash['my'] = 13
+    my_hash[:b] = 'name'
+    my_hash[404] = 'error'
+  end
+
+  describe '#[]=' do
+    context 'introduction of new elements key=>value' do
+      it 'return new elements in the hash' do
+        expect(my_hash[:c] = 10).to eq 10
+        expect(my_hash[1] = -3).to eq(-3)
+      end
+    end
+    it 'return key=>value' do
```

Comment:
here the example desc is wrong. `it` always refer to what you are describing. You should describe `[]=` method, but below you test `[]` method
If you want to test that once you assigned the value to a key you can then retrieve it, try that:
```
it 'assigns value to key' do
  expect(my_hash['new_key']).to eq null
  my_hash['new_key']  = 'some_value'
  expect(my_hash['new_key']).to eq 'some_value'
```